about/policy/sgid/index.md

```diff
@@ -1,116 +1,171 @@
 ---
-title: SGID Database Policies
+title: SGID Policies
 author:
   display_name: AGRC Office
   email: agrc@utah.gov
 tags:
   - policy
   - sgid
 categories: []
-date: 2019-12-04 10:01:27 -0500
+date: 2020-01-14 17:00
 ---
 
-The following policies apply to data in the internal SGID geodatabase, which is a curated database of state-wide datasets. It is maintained as an internal, AGRC-only Esri Enterprise Geodatabase and is available to everyone as the [Open SGID]({% link sgid/open-sgid/index.md %}) and as part of our [Open Data](https://opendata.gis.utah.gov) site.
+The policies on this page govern the way we interact with data in the [SGID]({% link sgid/index.html %}). Many of these policies also apply to external data being submitted to the SGID.
 
-1. TOC list
+- TOC list
 {:toc}
 
-## Adding New Datasets to the SGID Geodatabase
-{: .text-left }
+A Brief Word on Language
+{: style="
+    margin-bottom: .3em;
+    font-size: 1.4em;
+    text-align: left;
+    font-weight: 300;
+    color: #323232;
+    padding-top: .5em;
+    margin: 0 0 1em;
+    "}
 
-We strive to achieve and maintain a high standard of quality for the SGID. We use the following policies and procedures to guide this effort.
+We use the following terms intentionally on this page:
 
-### Qualifying Datasets
-{: .text-left }
+- **Must**: This is as close to a hard and fast rule as we get (short of a legislative "thou shalt").
```

Comment:
i go back and forth on the "short of a legislative "thou shalt"" footnote.   when i enter from a humor aspect, i like it, but when i enter from a serious aspect i'm a little scared. 

just my two cents

Reply:
In my head I was referencing our legislative mandates in state code, but that could be made more clear here. Maybe "This is as close to a hard and fast rule as we get (short of the legislative "thou shalt" rules in the legislation establishing AGRC)", with a link to https://gis.utah.gov/about/code/ ?